streams/us.m3u

```diff
@@ -248,8 +248,8 @@ http://ott-cdn.ucom.am/s65/index.m3u8
 https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CNCTV.us",County News Center (CNC TV) (San Diego CA) (720p)
 https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sdcounty/G0283_012/playlist.m3u8
-#EXTINF:-1 tvg-id="CNN.us",CNN
-https://16live00.akamaized.net/CNN/index.m3u8
+#EXTINF:-1 tvg-id="CNN.us",CNN [Geo-blocked]
+https://tve-live-ctl.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8
```

Comment:
Now doesn't work either.
<img width="686" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/da8175a8-6e0d-4e2b-9f37-7994a4705492">

Reply:
Did you try it via USA IP? It is geoblocked

Reply:
Yes